DashWallet/Sources/UI/DashPay/DWDashPayConstants.m

```diff
@@ -24,5 +24,6 @@
 
 NSInteger DW_MIN_USERNAME_LENGTH = 3;
 NSInteger DW_MAX_USERNAME_LENGTH = 24;
+BOOL MOCK_DASHPAY = YES; // TODO: remove once Platform is available
```

Comment:
I figure it's better to reference a single flag for any mocking code. Will be easier to turn off and remove once the platform is available.